src/views/Bond/BondHeader.jsx

```diff
@@ -1,7 +1,7 @@
 import { useState } from "react";
 import { NavLink, useHistory } from "react-router-dom";
 import BondLogo from "../../components/BondLogo";
-import AdvancedSettings from "./AdvancedSettings";
+import AdvancedSettings from "../BondV2/AdvancedSettings";
```

Comment:
@appleseed-iii - will this have any issues that you know of consolidating to a single advanced settings view, vs managing one for v1 and v2? It doesn't look like it should affect anything but just want to confirm.